epub34/a11y/index.html

```diff
@@ -106,17 +106,17 @@
 				},
 				preProcess:[inlineCustomCSS],
 				postProcess: [data_test_display, fixErrataField],
-				// otherLinks: [
-				//	{
-				//		key: "This document is also available in this non-normative format:",
-				//		data: [
-				//			{
-				//				value: "EPUB 3",
-				//				href: "./epub-a11y-12.epub"
-				//			}
-				//		]
-				//	}
-				//]
+				/* otherLinks: [
+					{
+						key: "This document is also available in this non-normative format:",
+						data: [
+							{
+								value: "EPUB 3",
+								href: "./epub-a11y-12.epub"
```

Comment:
shouldn't this be epub-a11y-111.epub?

Reply:
Yup, that code's commented out because we don't generate the epub version of the spec until the final stages of publishing the new version, so I wasn't paying attention to it.